location/templates/location.html

```diff
@@ -40,14 +45,90 @@
 {% endblock%}
 
 {% block content %}
-<h1>{{object.address}}, {{object.city}}, {{object.state}}, {{object.zipcode}}</h1>
+<div class="row">
+<div class="col col-sm-9">
+    <h1>{{object.address}}, {{object.city}}, {{object.state}}, {{object.zipcode}}</h1>
```

Comment:
nit: could we make this a little bit smaller? I think it'd fit better with the favorite/review button

Reply:
Changed to `<h3>`

Reply:
looks great! if you just wouldn't mind rebasing your branch I'll take one final peek and approve.

Reply:
rebased